documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -271,6 +272,31 @@ To explicitly preserve the case of the table and field names that are present in
 When this options is set, when an incoming event is for a topic called `orders`, and the destination database dialect is Oracle, the connector creates a table with the name `orders`, because the constructed SQL defines the name of the table as `"orders"`.
 Enabling quoting results in the same behavior when the connector creates column names.
 
+[[jdbc-connection-idle-timeouts]]
+=== Connection Idle Timeouts
+
+The JDBC sink connector for {prodname} leverages a connection pool to enhance performance.
+Connection pools are engineered to establish an initial set of connections, maintain a specified number of connections, and efficiently allocate them to the application as required.
```

Comment:
```suggestion
Connection pools are engineered to establish an initial set of connections, maintain a specified number of connections, and efficiently allocate connections to the application as required.
```
_"the application"_ being Debezium (and specifically the JDBC connector)?

Reply:
In this context, _application_ loosely means anything as I was describing connection pools and their purpose from a general purpose.